models/books.js

```diff
@@ -0,0 +1,14 @@
+const Books = (connection, Sequelize, Authors, PublishInfo, Artist, OriginalPublisher) => {
+  return connection.define('books', {
+    id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+    title: { type: Sequelize.STRING, allowNull: false },
+    authorId: { type: Sequelize.STRING, allowNull: false, references: { model: Authors, key: 'id' } },
+    publishInfoId: { type: Sequelize.STRING, allowNull: false, references: { model: PublishInfo, key: 'id' } },
+    // eslint-disable-next-line max-len
+    originalPublisherId: { type: Sequelize.STRING, allowNull: false, references: { model: OriginalPublisher, key: 'id' } },
```

Comment:
Ignoring max line is really only ok in tests, lines like this should just be reformatted, not ignored.

```JavaScript
originalPublisherId: { 
  type: Sequelize.STRING, allowNull: false, references: { model: OriginalPublisher, key: 'id' } 
},
```